.travis.yml

```diff
@@ -40,24 +38,26 @@ matrix:
   allow_failures:
     - php: 7.0
     - php: hhvm
-    - env: SYMFONY_VERSION=2.8.*@dev
-    - env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+    - env: SYMFONY_VERSION=2.8.*@dev SYMFONY_DEPRECATIONS_HELPER=strict
+    - env: SYMFONY_VERSION="3.0.x-dev as 2.8" SYMFONY_DEPRECATIONS_HELPER=strict
 
-before_script:
+before_install:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
   - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit:^4.8 fabpot/php-cs-fixer --no-update
-  - composer global update --prefer-dist --no-interaction
-  - if [ "$SYMFONY_VERSION" = "2.8.*@dev" ] || [ "$SYMFONY_VERSION" = "3.0.x-dev as 2.8" ]; then SYMFONY_DEPRECATIONS_HELPER=strict; fi;
+  - composer global require phpunit/phpunit:^4.8 --no-update
+  - if [ "$CS_FIXER" = "run" ]; then composer global require fabpot/php-cs-fixer --no-update; fi;
```

Comment:
A new verbose line for me.

Packages are cached so you will not save a lot of time with this condition.